Sources/PackageDescription/SupportedPlatforms.swift

```diff
@@ -10,8 +10,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-/// A platform that usually corresponds to an operating system such as
-/// iOS, macOS, or Linux.
+/// A platform that corresponds to an operating system such as iOS,
+/// macOS, or Linux.
```

Comment:
A platform is not _always_ an OS, non-OS examples listed in the type include `.driverKit` and `.wasi`.

Reply:
How about this: 
> A platform supported by Swift Package Manager.